CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ Categories: Added, Removed, Changed, Fixed, Nonfunctional, Deprecated
 
 ## Unreleased
 
+## 5.0.0 (2023.12.21)
+ - ci: use matrix for unit tests (#1395)
+ - Remove template_path config parameter (#1386)
```

Comment:
bumped to v5.0 for `template_path` removal however it looks release v5.0 also requires removal of `role_arn`...

https://app.circleci.com/pipelines/github/Sceptre/sceptre/2062/workflows/bc868609-26a6-441d-b992-32a9828d2484/jobs/11277
```
FAILED tests/test_stack.py::TestStack::test_role_arn__is_removed_on_removal_version - AssertionError: <function TestStack.test_role_arn__is_removed_on_removal_ve...
```